packages/manager/apps/web-ongoing-operations/src/pages/tracking/Tracking.tsx

```diff
@@ -0,0 +1,315 @@
+    queryFn: () => fetchDomainData(),
+  });
+
+  useEffect(() => {
+    if (tracking?.progress === 100) {
+      setTransfertFinalised(true);
+    }
+  }, [tracking?.progress]);
+
+  if (isLoading) {
+    return (
+      <div data-testid="listing-page-spinner">
+        <Loading />
+      </div>
+    );
+  }
+
+  return (
+    <BaseLayout
+      header={{
+        title: t('domain_operations_dashboard_title'),
+      }}
+    >
+      <SubHeader domain={domain} />
+
+      {transfertFinalised ? (
+        <div
+          className="flex flex-col items-center justify-center mt-16"
+          data-testid="done"
+        >
+          <div className="flex flex-col gap-y-3 items-center justify-center mb-4">
+            <OdsIcon name="check" className="success text-2xl mb-4" />
+            <OdsText preset={ODS_TEXT_PRESET.heading3}>
+              {t('tracking_transfert_finalized')}
+            </OdsText>
+          </div>
+          <OdsText className="mb-8" preset={ODS_TEXT_PRESET.paragraph}>
+            {t('tracking_transfert_sub_finalized')}
+          </OdsText>
+          <OdsButton
+            label={t('tracking_transfert_back_to_domain')}
+            onClick={() => navigate(urls.domain)}
+          />
+        </div>
+      ) : (
+        <div>
+          <OdsCard color={ODS_CARD_COLOR.neutral} className="w-3/4 mb-4">
+            <div className="px-3 py-3">
+              <div className="pl-4 flex items-center gap-x-2">
+                {[
+                  TrackingEnum.Initialisation,
+                  TrackingEnum.AskForAuthInfo,
+                ].includes(tracking?.currentStep.step) && (
+                  <OdsSpinner data-testid="init-spinner" size="xs" />
+                )}
+                {tracking?.progress > 0 && (
+                  <OdsIcon
+                    data-testid="init-check"
+                    name="circle-check"
+                    className="success"
+                  />
+                )}
+                <OdsText preset="heading-4">
+                  {t('tracking_transfert_first_step_operation_title')}
+                </OdsText>
+              </div>
+              {tracking?.currentStep.step === TrackingEnum.Initialisation && (
+                <div className="pl-5 mt-1" data-testid="init-section">
+                  <div>
+                    <OdsText preset={ODS_TEXT_PRESET.span}>
+                      {t('tracking_transfert_first_step_operation_create')}
+                    </OdsText>
+                    <OdsText
+                      preset={ODS_TEXT_PRESET.paragraph}
+                      className="block"
+                    >
+                      {t('tracking_transfert_first_step_operation_steps')}
+                    </OdsText>
+                    <OdsText preset={ODS_TEXT_PRESET.paragraph}>
+                      {`${t('tracking_transfert_last_update')} ${formatDate(
+                        tracking?.lastUpdateDate,
+                        l.language,
+                      )}`}
```

Comment:
Parameter should be included in translation

format should be wrapped in a useMemo

Also you have a `?` to check for nullable but if it is you will have a weird format